pkg/manager/task_identify.go

```diff
@@ -236,7 +236,16 @@ type scraperSource struct {
 }
 
 func (s scraperSource) ScrapeScene(sceneID int) (*models.ScrapedScene, error) {
-	return s.cache.ScrapeScene(s.scraperID, sceneID)
+	content, err := s.cache.ScrapeID(context.TODO(), s.scraperID, sceneID, models.ScrapeContentTypeScene)
```

Comment:
I think the `ScrapeScene` methods should be changed to accept a context and pass that through instead.

Reply:
Yes, this sounds like the right move to me. Done.